gui/slick/views/addShows_newShow.mako

```diff
@@ -7,7 +7,6 @@
 <script type="text/javascript" src="${srRoot}/js/qualityChooser.js?${sbPID}"></script>
 <script type="text/javascript" src="${srRoot}/js/addShowOptions.js?${sbPID}"></script>
 <script type="text/javascript" src="${srRoot}/js/rootDirs.js?${sbPID}"></script>
-<script type="text/javascript" src="${srRoot}/js/blackwhite.js?${sbPID}"></script>
```

Comment:
@p0psicles 

I don't think you meant to remove this.   This is the code for the black / white list for anime on the Add New Show page, not recommended.